### ISSUE TITLE
PointNet classifier config and how to get same result of SampleNet

### ISSUE THREAD

**Issue by chenxl124578** [ttchengab/MetaSampler#2]
Thanks to the author for sharing the code. That's a great work for Sampling task!

And I have some question want to ask author:
(1) Did you train classifier(PointNet) using the same code in Pointnet_Pointnet2_pytorch [https://github.com/yanx27/Pointnet_Pointnet2_pytorch](https://github.com/yanx27/Pointnet_Pointnet2_pytorch)?

(2) if (1) true, i  want to know how can you get the same result like in SampleNet's paper, which in your paper is named "Single" in "Table 1". Because the code in (1) is a little different from PointNet(Tensorflow source using in [SampleNet](https://github.com/itailang/SampleNet/tree/master/classification) ). Maybe the performance of PointNet(Pytorch) is better than PointNet(Tensorflow source code).

Looking forward to your reply, thank you!

**Comment by ttchengab** [comment]
Hi. Thank you for your appreciation for the work.

1) Yes, all classifiers are trained using the same code in Pointnet_Pointnet2_pytorch.

2) As mentioned in our paper, all the classifiers are trained to roughly 89%~90% when we freeze to train SampleNet (similar to the original PointNet proposed 89.2%). We tested with a single classifier to train SampleNet and it yielded similar results to the ones reported in SampleNet itself, and joint training does show an increase in accuracy compared using a single classifier. To eliminate any discrepancies, we take their results (SampleNet) for our comparison in classification. 

I hope this helps and thanks again!

**Comment by chenxl124578** [comment]
Thanks for your reply!
